docs/concepts/what-is-stash/overview.md

```diff
@@ -13,7 +13,7 @@ section_menu_id: concepts
 ---
 # Stash
 
- Stash by AppsCode is a Kubernetes operator for [restic](https://restic.net). If you are running production workloads in Kubernetes, you might want to take backup of your disks. Traditional tools are too complex to setup and maintain in a dynamic compute environment like Kubernetes. `restic` is a backup program that is fast, efficient and secure with few moving parts. Stash is a CRD controller for Kubernetes built around `restic` to address these issues. Using Stash, you can backup Kubernetes volumes mounted in following types of workloads:
+ Stash by AppsCode is a Kubernetes operator for [backup](https://backup.net). If you are running production workloads in Kubernetes, you might want to take backup of your disks. Traditional tools are too complex to setup and maintain in a dynamic compute environment like Kubernetes. `backup` is a backup program that is fast, efficient and secure with few moving parts. Stash is a CRD controller for Kubernetes built around `backup` to address these issues. Using Stash, you can backup Kubernetes volumes mounted in following types of workloads:
```

Comment:
Use: **restic** instead of **backup**
https://restic.net instead of https://backup.net